public/v2/apps/mautic.json

```diff
@@ -0,0 +1,77 @@
+{
+  "captainVersion": "2",
+  "displayName": "Mautic beta",
+  "description": "Mautic is an open source marketing automation platform.",
+  "dockerCompose": {
+    "version": "2",
+    "services": {
+      "$$cap_appname-db": {
+        "documentation": "Taken from https://hub.docker.com/_/percona",
+        "image": "percona/percona-server:$$cap_percona_version",
+        "notExposeAsWebApp": "true",
+        "volumes": [
+          "$$cap_appname-db-data:/var/lib/mysql"
+        ],
+        "ports": [
+          "3306:3306"
```

Comment:
You don't need to expose the port to the outside world. You can remove this.

Reply:
agreed, solved